### DISCUSSION TITLE
[QUESTION] default RTE toolbar appearance

### DISCUSSION THREAD

**Discussion by KaoDeo** [GrapesJS/grapesjs#4866] [Q&A] [answered]
Hi guys,

I am currently using the latest version (0.20.21) of the `grapesjs-preset-newsletter` plugin, but I noticed that the rich text editor toolbar appears differently compared to the toolbar on the demo newsletter website. I was expecting the default view of the toolbar to match the one on the demo. Can you help me understand how I can achieve this? Am I missing something?

My toolbar:
<img width="150" alt="image" src="https://user-images.githubusercontent.com/91371368/214518721-8bd021bc-d808-4acd-ac66-a2ebc62fdb15.png">

Demo toolbar:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/91371368/214518498-2e9a4887-993c-4fb8-9790-1226c2f1cd37.png">

Thanks 🙏

**Comment by artf** [comment, marked as answer]
That because of the `grapesjs-plugin-ckeditor` [used in the demo](https://github.com/artf/grapesjs/blob/cf737a7a26a03d307abe721692db6d4c568d947d/demo-newsletter-editor.html#L561).
